blockchain/state_manager/src/lib.rs

```diff
@@ -176,7 +214,7 @@ where
 
             let block_headers = tipset.blocks();
             // generic constants are not implemented yet this is a lowcost method for now
-            let cid_pair = self.compute_tipset_state(&block_headers)?;
+            let cid_pair = self.compute_tipset_state(&block_headers, Some(|_, _, _| Ok(())))?;
```

Comment:
what's the point of that callback? Seems it should be None?

Reply:
Yes this should be none, think I copied directly from lotus. They have a nullable callback but implemented this